pages/common/aws-codecommit.md

```diff
@@ -0,0 +1,12 @@
+# aws codecommit
+
+> AWS CodeCommit is a secure, highly scalable, managed source control service that hosts private Git repositories.
```

Comment:
We try to use an objective description here
```suggestion
> AWS CodeCommit is a managed source control service that hosts private Git repositories.
```

Reply:
@navarroaxel @kbdharun  I have pushed the change as you want changed the description to - 
AWS CodeCommit is a managed source control service that hosts private Git repositories.
Can you check and confirm please.